store/sqlstore/post_store.go

```diff
@@ -1321,7 +1321,7 @@ func (s *SqlPostStore) determineMaxPostSize() int {
 				table_name = 'posts'
 			AND	column_name = 'message'
 		`); err != nil {
-			mlog.Error(utils.T("store.sql_post.query_max_post_size.error") + err.Error())
+			mlog.Error("Unable to determine the maximum supported post size", mlog.Err(err))
```

Comment:
Can you also remove the line that refers to `store.sql_post.query_max_post_size.error` in `i18n/en.json`? I'm definitely a fan of removing unnecessary translations, but we don't want to leave that behind

Reply:
only from en.json or all locales. Its in multiple locale files

Reply:
removed from all the locales

Reply:
It should be removed only from the `i18n/en.json` file, the rest of the files are update through our translation process.